testsuite/MDAnalysisTests/coordinates/base.py

```diff
@@ -208,7 +209,14 @@ def reader(ref):
         reader.add_auxiliary('lowf', ref.aux_lowf, dt=ref.aux_lowf_dt, initial_time=0, time_selector=None)
         reader.add_auxiliary('highf', ref.aux_highf, dt=ref.aux_highf_dt, initial_time=0, time_selector=None)
         return reader
-
+    
+    @staticmethod
+    @pytest.fixture()
+    def transformed(ref):
+        transformed = ref.reader(ref.trajectory)
+        transformed.add_transformations([translate(1,1,1)])
```

Comment:
It would be a good idea to have more than 1 transformation in your workflow. If, by any bug, the loop is broken when applying the transformations, it will be invisible if you have only one transformation. If you use translate again, make sure it is possible to identify multiple run of each transformation. For instance, you could use `translate(0, 0, 0.33)` .